packages/core/database/lib/migrations/index.js

```diff
@@ -26,6 +27,7 @@ const createUmzugProvider = db => {
   const migrationDir = path.join(strapi.dirs.root, 'database/migrations');
 
   fse.ensureDirSync(migrationDir);
+  if (fs.readdirSync(migrationDir).length === 0) fse.ensureFile(migrationDir + '/.gitkeep');
```

Comment:
fs-extra exports the same functions as  fs (just no re documented) so you can use fse.readdirSync the same way too. no need for fs at all.

Reply:
Ah alright wasn't aware of that since their documentation only shows 3 functions.

I'll swap it to fse then after lunch.